### PR TITLE
Fixed csv time parse bug, and added updated spring 2021 csv

### LINKED ISSUE #309: Bug — Incorrect Class Time for ECSE-2050
**Describe the bug**
ECSE-2050 has incorrect class time recorded, with the 8:00 pm time slot for Tuesday replaced with 8:00 am. 

**To Reproduce**
1. Go to the main page for the Spring 2021 semester.
2. Pull up the ECSE-2050 course.

**Expected behavior**
Time on Tuesday should be from 8:00 pm to 9:50 pm, rather than from 8:00 am to 9:50 pm. 

**Screenshots**
image
image

**Additional context**
EDIT: Further investigation reveals that it happens with any class with an 8:00pm class start time.

### PR DESCRIPTION
**Pull request by marchdan**
**Issue**

Closes #309 

**Test Procedure**

Show procedure to test functionality with a clear procedure

*Example:*
> 1. Compile a new Spring 2021 CSV
> 2. Upload the CSV in the admin panel of your test enviroment
> 3. Check that ECSE 2050 & ECSE 2100 display correctly

### CODE REVIEW

**Review by 123joshuawu — APPROVED** [review]

File: rpi_data/modules/fetch_sis_course_info.py [hunk 1]
```diff
@@ -125,8 +125,13 @@ def run(self):
...
                     else:
-                        col = info[6].find(':')
-                        if (int(info[6][:col]) < 8) or (int(info[6][:col]) == 12):
+                        col1 = info[6].find(':')
+                        col2 = info[7].find(':')
+                        if int(info[6][:col1]) == 12:
+                            info[6] += "PM"
+                        elif int(info[7][:col2]) == 12:
+                            info[6] += "AM"
+                        elif int(info[6][:col1]) < int(info[7][:col2]):
```
Comment (jshom) [inline rpi_data/modules/fetch_sis_course_info.py:134]: Can we rename this and add some comments here? I'm having trouble following the logic here.
Also might be nice to alias or rename some variables here

Comment (marchdan) [reply rpi_data/modules/fetch_sis_course_info.py:134]: I can add comments, info is the name of the course information, so there can't really be much more descriptive for an all-encompassing variable, but I'll get some good comments in there

**Comment by marchdan** [comment]
@jshom anything else needed?

**Comment by 123joshuawu** [comment]
@marchdan er did u mean to merge into master

**Pull request closed without merging**